docs/howto/manage-charms.rst

```diff
@@ -11,10 +11,11 @@ Manage charms
 Initialise a charm
 ------------------
 
-.. note::
+.. admonition:: Best practice
```

Comment:
This is in the how-to but it doesn't change the content, it just adjusts it to be a best practice rather than only a note.